src/screens/Portfolio/AnimatedTopBar.js

```diff
@@ -16,14 +16,30 @@ import BalanceHeader from "./BalanceHeader";
 import HeaderErrorTitle from "../../components/HeaderErrorTitle";
 import HeaderSynchronizing from "../../components/HeaderSynchronizing";
 
-class AnimatedTopBar extends PureComponent<{
-  scrollY: AnimatedValue,
-  portfolio: Portfolio,
-  counterValueCurrency: Currency,
-  pending: boolean,
-  error: ?Error,
-  navigation: *,
-}> {
+class AnimatedTopBar extends PureComponent<
+  {
+    scrollY: AnimatedValue,
+    portfolio: Portfolio,
+    counterValueCurrency: Currency,
+    pending: boolean,
+    error: ?Error,
+    navigation: *,
+  },
+  { skipRender: boolean },
```

Comment:
can you explain / comment in this code, why we need this variable?

skipRender is not enough explicit to understand what it does (maybe should rename it too. it feels like you are actually skipping the render()ing which is not the case)

Reply:
bad naming but the thing is, we animate the opacity of the bar. However, the bar while invisible is still capturing all the touch events. This flag will kill the pointer events while opacity is 0, using the offset. Renaming it to "ignorePointerEvents" would make more sense